src/analysis/retail/deathknight/blood/modules/Abilities.ts

```diff
@@ -35,14 +35,14 @@ class Abilities extends CoreAbilities {
       },
       {
         spell: TALENTS.ANTI_MAGIC_ZONE_TALENT.id,
-        enabled: combatant.hasTalent(TALENTS.ANTI_MAGIC_ZONE_TALENT),
         buffSpellId: SPELLS.ANTI_MAGIC_ZONE_TALENT_BUFF.id,
         category: SPELL_CATEGORY.DEFENSIVE,
-        cooldown: 120,
         gcd: {
           base: 1500,
         },
+        cooldown: combatant.hasTalent(TALENTS.ASSIMILATION_TALENT) ? 120 : 180,
```

Comment:
isn't AMZ now a 4 minute cooldown (reduced to 3 with Assimilation)? so it should be:

```suggestion
        cooldown: combatant.hasTalent(TALENTS.ASSIMILATION_TALENT) ? 180 : 240,
```

and the same for the other specs